express-postgres/server.ts

```diff
@@ -0,0 +1,84 @@
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+import pg from 'pg';
+
+const app = express();
+
+app.use(errorMiddleware);
```

Comment:
I believe this is the problem. It must be the _last_ middleware in your file. This is why you are not seeing error messages in the terminal or getting 500 responses in the http.
So move it down to the bottom, just before the call to `app.listen`.